src/reporters.ts

```diff
@@ -30,14 +30,22 @@ export class JestReporter implements jest.Reporter {
     }
     static initializeKelonio(): void {
         const state = new BenchmarkFileState();
-        state.write({});
+        if(process.env.KELONIO_KEEP_STATE_AT_START === "true") {
+            state.append({});
+        } else {
+            state.write({});
+        }
```

Comment:
If we implement Jest's `onRunStart()` and move this code there, we should be able to use the regular reporter configuration instead of an environment variable. Would you mind giving that a try?

Reply:
Sure, I'll try that out and message back on here the results.